device/bgp/bgpv4.yaml

```diff
@@ -7,16 +7,15 @@ components:
       description: >-
         Container for emulated BGPv4 peers and routes.
       type: object
+      required: [dut_address]
       properties:
         local_address:
```

Comment:
removal of default should make this a required property

Reply:
Most of tgens would take local address from ipv4 stack and populate the local address. removal of default would make it to None and the tgen default values are applied.

Reply:
`local_address` is now a required field